command/operator_generate_root.go

```diff
@@ -74,11 +74,16 @@ Usage: vault operator generate-root [options] [KEY]
 
   Start a root token generation:
 
+      $ vault operator generate-root -init
```

Comment:
Given that this will generate an OTP code for you, it feels like it might be more appropriate to list it first, before `Generate an OTP code for the final token`, and clarify that it will generate an OTP code at the same time. That might make it a bit clearer that the `Generate an OTP code for the final token` step isn't necessary if you go this route.

The learn guide demonstrates this flow pretty well. https://learn.hashicorp.com/tutorials/vault/generate-root

Reply:
You're right, that's a good idea

Reply:
I modified this, added some clarifications